components/kyma-environment-broker/internal/provider/openstack_provider.go

```diff
@@ -0,0 +1,61 @@
+package provider
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/ptr"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+)
+
+const (
+	DefaultOpenStackRegion = "eu-de-1"
+)
+
+type OpenStackInput struct {
+}
+
+func (p *OpenStackInput) Defaults() *gqlschema.ClusterConfigInput {
+	return &gqlschema.ClusterConfigInput{
+		GardenerConfig: &gqlschema.GardenerConfigInput{
+			DiskType:       ptr.String("pd-standard"),
+			VolumeSizeGb:   ptr.Integer(30),
+			MachineType:    "m1.large",
```

Comment:
To match Azure, GCP and AS production plans, we could use m2.xlarge (8 CPU / 16 GB RAM) or m1.2xlarge (8 CPU / 32 GB RAM). But on CCloud I'm a bit worried about node unavailability more than with other providers